Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -103,5 +105,76 @@ public TestsRepository(TrappistDbContext dbContext, IGlobalUtil util)
             await _dbContext.SaveChangesAsync();
         }
         #endregion
+        #region Category selection
+        public async Task<TestAC> GetTestDetailsByIdAsync(int testId)
+        {
+            var testAcObject = new TestAC();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable 'testAcObject'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1854)